client/cypress/integration/visualizations/pivot_spec.js

```diff
@@ -99,6 +99,10 @@ describe("Pivot", () => {
         .focus()
         .type(" UNION ALL {enter}SELECT 'c' AS stage1, 'c5' AS stage2, 55 AS value");
 
+      // wait for the query text change to propagate (it's debounced in QuerySource.jsx)
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(200);
```

Comment:
For tasks that need to be executed, you may wish to increase the waiting time. My suggestion is `500 or 1000`.
`200` doesn't seem to do much. Just my opinion.

Reply:
Oops, I merged it before seeing that.

I guess if we keep on seeing failures here then we can raise that limit. :smile:

Reply:
> Oops, I merged it before seeing that.
> 
> I guess if we keep on seeing failures here then we can raise that limit. 😄

Yeah. It doesn't matter.

Reply:
If the reason is really the debounce issue, 200 should be perfectly fine (2x the actual debounce time). It's usually a non-practice to do those arbitrary waiting times, it's just that for this case I had no clue what I could use in the UI to assure the state was updated 😕.

But let's see if it helps

Reply:
> If the reason is really the debounce issue, 200 should be perfectly fine (2x the actual debounce time). It's usually a non-practice to do those arbitrary waiting times, it's just that for this case I had no clue what I could use in the UI to assure the state was updated 😕.
> 
> But let's see if it helps

Good job!